src/nectarchain/dqm/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .db_utils import DQMDB
 from .dqm_summary_processor import DQMSummary
+from .pixel_participation import PixelParticipationHighLowGain
 
-__all__ = ["DQMDB", "DQMSummary"]
+__all__ = ["DQMDB", "DQMSummary", "PixelParticipationHighLowGain"]
```

Comment:
What is the reason why `PixelParticipationHighLowGain` should be exposed, and not the other DQM classes ? We should take care of the fact that PR #148 already exposes all other DQM classes. We may hit a conflict at some point, which should be easy to solve.